tests/fail-with-invalid-chars-in-message/HelloWorld.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+function helloWorld()
+{
+    $invalidChars = [
+        "\u{7F}", // Delete
+        "\xFF\xFF\xFF\xFF\xFF\xFF", // Invalid UTF-8
```

Comment:
Is it important to have this repetition? Or we could stick to one?


```suggestion
        "\xFF", // Invalid UTF-8
```

Reply:
UTF-8 uses bit 8 in `\xFF` to say: the next byte adds to this character. So whatever follows a single `\xFF` is valid UTF-8. To become invalid, a sequence of 6 `\xFF` bytes is required, as UTF-8 may have up to 6 bytes but never 7.